src/Calculate.ts

```diff
@@ -839,50 +840,48 @@ export const calculateOffline = (forceTime = 0) => {
         quarks: quarkHandler().gain //Calculate this after the fact
     };
 
-    //Reset Stuff lmao!
-    addTimers('prestige', timeAdd);
-    addTimers('transcension', timeAdd);
-    addTimers('reincarnation', timeAdd);
     addTimers('ascension', timeAdd);
     addTimers('quarks', timeAdd);
 
     player.prestigeCount += resetAdd.prestige;
     player.transcendCount += resetAdd.transcension;
     player.reincarnationCount += resetAdd.reincarnation;
-
+    
     timerAdd.ascension = player.ascensionCounter - timerAdd.ascension
     timerAdd.quarks = quarkHandler().gain - timerAdd.quarks
 
-    document.getElementById('offlineAscensionTimerNumber').textContent = format(timerAdd.ascension, 2, true)
-    document.getElementById('offlineQuarkCountNumber').textContent = format(timerAdd.quarks, 0, true)
+    //200 simulated all ticks [June 18, 2021]
+    function resourceSimulator() {
```

Comment:
please move this function into the interval above & make it an arrow function for consistency

Reply:
Not very familiar with arrow functions (I had just modified the function that already existed), but hopefully this is correct (it seems to run without issue, but want it to be consistent).  

If there is a better way to do this let me know and I'll make the adjustments.

```js
const runOffline = interval(() => resourceSimulator(), 0);
//200 simulated all ticks [July 12, 2021]
const resourceSimulator = () => {
...
```

Reply:
since resourceSimulator isn't used anywhere else in the code, it could be simplified to something like

```js
const runOffline = interval(() => {
    // do resourceSimulator stuff here
}, 0);
```

Reply:
Ahhh makes that sense, sorry for the trouble.

Made the suggested change, thanks for the feedback/help Khafra

Reply:
quarks please